src/pudl/metadata/resources/__init__.py

```diff
@@ -199,28 +193,33 @@
             "boiler_fuel_code_6",
             "boiler_fuel_code_7",
             "boiler_fuel_code_8",
+            "waste_heat_input_mmbtu_per_hour",
+            "wet_dry_bottom",
+            "fly_ash_reinjection",
+            "hrsg",
+            "max_steam_flow_thousand_pounds_per_hour",
```

Comment:
There are some other thousands of pounds per hour (and pounds fields) if you search for `lbs` in `pudl.metadata.fields`. It looks like in general we've used the abbreviation `lbs` for pounds. We should probably use the same naming for all the thousands of pounds per hour fields, but I don't feel strongly about what it is.